pkg/controllers/plugin/helm_chart_testing.go

```diff
@@ -93,3 +93,40 @@ func (r *PluginReconciler) reconcileHelmChartTest(ctx context.Context, plugin *g
 
 	return nil, nil
 }
+
+func extractErrorsFromTestPodLogs(testPodLogs string) string {
+	var errors []string
+	var errorBlock strings.Builder
+
+	scanner := bufio.NewScanner(strings.NewReader(testPodLogs))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+		fmt.Println(line)
```

Comment:
```suggestion
		fmt.Println(line)
```

Reply:
Thank you. fixed in 5f3e79a528b9277020d3e55b887bdc4199499bd2